ci-operator/step-registry/baremetal/lab/pre/load-balancer/baremetal-lab-pre-load-balancer-commands.sh

```diff
@@ -90,6 +105,16 @@ listen ingress-router-443
     mode tcp
     balance source
 $INGRESS443
+listen ironic-api-6385
+    bind *:6385
+    mode tcp
+    balance source
+$IRONICSRV
+listen inspector-api-5050
+    bind *:5050
+    mode tcp
+    balance source
+$INSPECTORSRV
```

Comment:
@zhaozhanqi setting the external load balancer IP let the installer set the Ironic IP to that external IP too, meaning that the external LB should expose the ironic APIs too (towards the bootstrap IP first and the master nodes later probably).

See this excerpt from the computed terraform vars:
```
sh-5.1$ cat terraform.platform.auto.tfvars.json  
...
  "ironic_uri": "http://10.10.137.98:6385/v1",
  "inspector_uri": "http://10.10.137.98:5050/v1",
  "bootstrap_os_image": "/alabama/.cache/openshift-installer/image_cache/rhcos-415.92.202311241643-0-qemu.aarch64.qcow2",
  "bridges": [
    {
      "mac": "52:54:00:00:00:49",
      "name": "baremetal"
    },
    {
      "mac": "52:54:00:d2:f3:6a",
      "name": "br--op-sfh1mjrk"
    }
  ],
  "ironic_username": "bootstrap-user",
.....
...
```
- The ironic uri is set to the API VIP, which is the external one.

- Is exposing the ironic APIs through the external lb acceptable in this network scenario?
- Does the internal lb do the same? I.e., Can users reach BM_NETWORK_API_VIP:6385 in non-external lb scenarios? @derekhiggins could give some hint maybe. 

In case this is ok, we will need to document this requirement (the way i did it was just the fastest to test/debug it, I'll consolidate later).